sisl/geometry.py

```diff
@@ -2280,61 +2287,87 @@ def rotate(self, angle, v, origin=None, atoms: Optional[AtomsArg]=None, only='ab
              argument to use radians.
         v     : int or str or array_like
              the normal vector to the rotated plane, i.e.
-             v = [1,0,0] will rotate the ``yz`` plane
+             v = [1,0,0] will rotate around the :math:`yz` plane.
+             If a str it refers to the Cartesian direction (xyz), or the
+             lattice vectors (abc). Providing several is the combined direction.
         origin : int or array_like, optional
              the origin of rotation. Anything but [0, 0, 0] is equivalent
              to a `self.move(-origin).rotate(...).move(origin)`.
              If this is an `int` it corresponds to the atomic index.
         atoms : int or array_like, optional
              only rotate the given atomic indices, if not specified, all
              atoms will be rotated.
-        only : {'abc+xyz', 'xyz', 'abc'}
-             which coordinate subject should be rotated,
-             if ``abc`` is in this string the cell will be rotated
-             if ``xyz`` is in this string the coordinates will be rotated
         rad : bool, optional
              if ``True`` the angle is provided in radians (rather than degrees)
+        what : {'xyz', 'abc', 'abc+xyz'}
```

Comment:
Is it clear from this line that one can also specify `what="x"` as in your example below? Or combinations  like `what="xy+b"`?

Reply:
thanks, I have amended, could you have a look?

Reply:
Looks fine